ra_aid/__main__.py

```
@@ -788,6 +806,130 @@ def build_status():
+    config_repo.set("web_research_enabled", args.web_research)
+    config_repo.set("search_enabled", args.search)
+    config_repo.set("hil", args.hil)
+    config_repo.set("test_cmd_timeout", args.test_cmd_timeout)
+    config_repo.set("max_test_cmd_retries", args.max_test_cmd_retries)
+    
+    # Initialize MCP-Use integration if enabled
+    mcp_use_client_instance = None
+    if hasattr(args, 'mcp_use_config') and args.mcp_use_config:
+        try:
+            from ra_aid.utils.mcp_use_client import MCPUseClientSync
+            logger.info(f"Attempting to initialize MCP-Use client...")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)